de.js

```diff
@@ -0,0 +1,334 @@
+    connect_with_x: `Mit 𝕏 verbinden`,
+    start_receive_tips_twitch: `Erhalte Tips über Twitch`,
+    visit_mydogetv_twitch: `Besuche MyDogeTV auf Twitch für weitere Infos`,
+    connect_with_twitch: `Mit Twitch verbinden`,
+    screen_not_exist: "Dieser Bildschirminhalt existiert nicht.",
+    go_to_home_screen: `Gehe zum Start-Bildschirm`,
+    gift_cards: `Geschenkkarten`,
+    backup_wallet: `Erstelle ein Backup deiner neuen Wallet`,
+    backup_required: `Zur Sicherung deines Guthabens ist ein Backup erforderlich`,
+    why_backups_important: `Warum sind Backups wichtig?`,
+    learn_about_backups: `Tippe hier um mehr über Backups zu erfahren`,
+    backup_to: `Backup per`,
+    save_wallet_to_cloud: ` Speichere deine Wallet in der Cloud und sichere sie mit einem Passwort`,
+    write_on_paper: `Auf Papier schreiben`,
+    write_down_seed_phrase: `Schreibe deine Seed-Phrase auf oder speichere sie in einem Passwort-Manager`,
+    why_backups_important_description: `Selbstverwaltung bedeutet, dass niemand sonst Zugriff auf dein Guthaben hat, nicht einmal das Team von MyDoge. Ein Backup ist die einzige Möglichkeit deine Wallet wiederherzustellen solltest du dein Gerät verlieren, es beschädigt, gestohlen oder gelöscht werden.`,
+    choose_password: `Wähle ein Passwort`,
+    use_memorable_password: `Verwende bitte ein Passwort, dass du dir gut merken kannst. Längere Passwörter sind sicherer.`,
+    more_secure_password: `Verwende bitte ein Passwort, dass du dir gut merken kannst. Längere Passwörter sind sicherer.`,
+    lose_password_lose_backup: `Wenn du dieses Passwort verlierst, verlierst du Zugriff auf dein Wallet-Backup.`,
+    password: `Passwort`,
+    confirm_password: `Passwort bestätigen`,
+    confirm_backup: `Backup bestätigen`,
+    overwrite_backup_title: `Vorhandenes Backup überschreiben?`,
+    existing_wallet_overwrite_prompt: `Du hast bereits ein Backup einer Wallet in der Cloud. Willst du es überschreiben?`,
+    overwrite_backup: `Backup überschreiben`,
+    import_existing_wallet: `Bestehende Wallet importieren`,
+    unknown_cloud_error: `Unbekannter Cloud-Fehler`,
+    unknown_cloud_error_body: `Beim Backup in die Cloud gab es einen unbekannten Fehler.`,
+    try_again_or_notify_support: `Bitte probiere es noch einmal oder benachrichtige den MyDoge-Support.`,
+    enter_recovery_password: `Passwort zur Wiederherstellung eingeben`,
+    enter_cloud_password_description: `Gib das Passwort ein, dass du zum Speichern deiner Seed-Phrase in der Cloud verwendet hast`,
+    import_wallet: `Wallet importieren`,
+    imported_wallet_title: `Du hast deine Wallet importiert`,
+    imported_wallet_success: `Du hast deine Wallet erfolgreich importiert!`,
+    secure_new_wallet: `Sichere deine neue Wallet ab`,
+    set_pin: `6-stellige PIN vergeben`,
+    reenter_pin: `6-stellige PIN wiederholen`,
+    pin_payment_note: "Du benötigst deine PIN um Zahlungen zu senden",
+    use_biometric: `Fingerabdruck/Gesichtserkkennung benutzen`,
```

Comment:
'Gesichtserkkennung' has a 'k' too much

```suggestion
    use_biometric: `Fingerabdruck/Gesichtserkennung benutzen`,
```